backend/src/api/Data/UsersDbContext.cs

```diff
@@ -4,10 +4,10 @@
 
 namespace api.Data;
 
-public class UsersDbContext : IdentityDbContext<UserModel>
+public class UsersDbContext : DbContext
```

Comment:
As this is no longer IdentityDbContext, I believe it would be wise to rename it to `ApplicationDbContext`

Reply:
Fixed, renamed to `ApplicationDbContext`.